types/src/Experiment/enums.ts

```diff
@@ -19,6 +19,10 @@ export enum CONDITION_ORDER {
 export enum ASSIGNMENT_ALGORITHM {
   RANDOM = 'random',
   STRATIFIED_RANDOM_SAMPLING = 'stratified random sampling',
+
```

Comment:
I am not sure about this! In the frontend, the user will be able to select additional algorithms. We will be able to add them as we test and support them, mooclet has a dozen or so algorithms we could potentially support. So I guess I need to add them to this enum instead of elsewhere with other Mooclet types?